networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/MultiaddrUtil.java

```diff
@@ -30,19 +30,29 @@ public static Multiaddr fromDiscoveryPeer(final DiscoveryPeer peer) {
     return fromInetSocketAddress(peer.getNodeAddress(), getNodeId(peer));
   }
 
+  public static Multiaddr fromDiscoveryPeerAsUdp(final DiscoveryPeer peer) {
+    return addPeerId(fromInetSocketAddress(peer.getNodeAddress(), "udp"), getNodeId(peer));
+  }
+
   public static Multiaddr fromInetSocketAddress(final InetSocketAddress address) {
+    return fromInetSocketAddress(address, "tcp");
+  }
+
+  public static Multiaddr fromInetSocketAddress(
```

Comment:
nit: I'm be tempted to make this private and just have the two `fromDiscoveryPeer` methods use it internally.